webui/react/src/components/kit/Select.tsx

```diff
@@ -0,0 +1,168 @@
+import { Select as AntdSelect } from 'antd';
```

Comment:
regresiion: The width looks different from before

<img width="452" alt="image" src="https://user-images.githubusercontent.com/109113616/222298592-1d983279-279f-441f-9f32-86ea488b7693.png">

<img width="325" alt="image" src="https://user-images.githubusercontent.com/109113616/222298672-4dc4207f-59b1-4822-9480-2926ac1ea80e.png">

Reply:
@keita-determined this is intended, we used to allow custom widths, the goal of my implementation was to standardized the component. If we do care about differing width sizes we could introduce some concept of `small`, `medium`, `large` etc.